http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/AKSInterpreter.scala

```diff
@@ -110,64 +102,81 @@ class AKSInterpreter[F[_]](config: AKSInterpreterConfig,
         )
         .run(authContext.copy(namespace = config.aadPodIdentityConfig.namespace))
 
+      // Create relay hybrid connection pool
+      hcName = RelayHybridConnectionName(params.appName.value)
+      relayPrimaryKey <- azureRelayService.createRelayHybridConnection(params.landingZoneResources.relayNamespace,
+                                                                       hcName,
+                                                                       params.cloudContext
+      )
+      relayEndpoint = s"https://${params.landingZoneResources.relayNamespace.value}.servicebus.windows.net/"
+
       // Deploy setup chart
       _ <- helmClient
         .installChart(
           Release(s"${app.release.asString}-setup-rls"),
           config.terraAppSetupChartConfig.chartName,
           config.terraAppSetupChartConfig.chartVersion,
-          org.broadinstitute.dsp.Values(
-            s"cloud=azure,serviceAccount.name=${ksaName.value}"
+          buildSetupChartOverrideValues(app.release,
+                                        app.samResourceId,
+                                        ksaName,
+                                        params.landingZoneResources.relayNamespace,
+                                        hcName,
+                                        relayPrimaryKey
           ),
           true
         )
         .run(authContext)
 
-      // Create relay hybrid connection pool
-      hcName = RelayHybridConnectionName(params.appName.value)
-      primaryKey <- azureRelayService.createRelayHybridConnection(landingZoneResources.relayNamespace,
-                                                                  hcName,
-                                                                  params.cloudContext
-      )
-
       // Resolve pet managed identity in Azure
       msi <- buildMsiManager(params.cloudContext)
       petMi <- F.delay(
         msi.identities().getById(petEmail.value)
       )
 
       // Assign the pet managed identity to the VM scale set backing the cluster node pool
-      _ <- assignVmScaleSet(landingZoneResources.clusterName, params.cloudContext, petMi)
-
-      // Build values and install chart
-      values = buildCromwellChartOverrideValues(app.release,
-                                                params.cloudContext,
-                                                app.samResourceId,
-                                                landingZoneResources,
-                                                hcName,
-                                                primaryKey,
-                                                petMi
-      )
-      _ <- helmClient
-        .installChart(
-          app.release,
-          app.chart.name,
-          app.chart.version,
-          values,
-          createNamespace = true
-        )
-        .run(authContext)
+      _ <- assignVmScaleSet(params.landingZoneResources.clusterName, params.cloudContext, petMi)
+
+      // Deploy app chart
```

Comment:
Looking forward to when we have more app types on Azure (e.g. after we split out WDS).

I'm treating the storage container as optional, and only required for Cromwell app at the moment.